website/pages/en/release-notes/graphql-validations-migration-guide.mdx

```diff
@@ -0,0 +1,540 @@
+---
+title: GraphQL Validations migration guide
+---
+
+Soon `graph-node` will support 100% coverage of the [GraphQL Validations specification](https://spec.graphql.org/June2018/#sec-Validation).
+
+Previous versions of `graph-node` did not support all validations and provided more graceful responses - so, in cases of ambiguity, `graph-node` was ignoring invalid GraphQL operations components.
+
+GraphQL Validations support is the pillar for the upcoming new features and the performance at scale of The Graph Network.
+
+It will also ensure determinism of query responses, a key requirement on The Graph Network.
+
+**Enabling the GraphQL Validations will break some existing queries** sent to The Graph API.
+
+To be compliant with those validations, please follow the migration guide.
+
+> ⚠️ If you do not migrate your queries before the validations are rolled out, they will return errors and possibly break your frontends/clients.
+
+# Migration guide
+
+You can use the CLI migration tool to find the issue in your GraphQL operations and fix them. Alternatively you can update the endpoint of your GraphQL client to use the `https://api-next.thegraph.com/subgraphs/name/$GITHUB_USER/$SUBGRAPH_SLUG` endpoint. Testing your queries against this endpoint will help you find the issues in your queries.
+
+> Not all subgraphs will need to be migrated, if you are using [GraphQL ESlint](https://the-guild.dev/graphql/eslint/docs) or [GraphQL Code Generator](https://the-guild.dev/graphql/codegen), they already ensure that your queries are valid.
+
+## Migration CLI tool
+
+**Most of the GraphQL operations errors can be found in your codebase ahead of time.**
+
+For this reason, we provide a smooth experience for validating your GraphQL operations during development or in CI.
+
+[`@graphql-validate/cli`](https://github.com/saihaj/graphql-validate) is a simple CLI tool that helps validate GraphQL operations against a given schema.
+
+### **Getting started**
+
+You can run the tool as follows:
+
+```bash
+npx @graphql-validate/cli -s https://api-next.thegraph.com/subgraphs/name/$GITHUB_USER/$SUBGRAPH_SLUG -o *.graphql
+```
+
+**Notes:**
+
+- Set or replace $GITHUB_USER, $SUBGRAPH_SLUG with the appropriate values. Like: [`artblocks/art-blocks`](https://api.thegraph.com/subgraphs/name/artblocks/art-blocks)
+- The preview schema URL (https://api-next.thegraph.com/) provided is heavily rate-limited and will be sunset once all users have migrated to the new version. **Do not use it in production.**
+- Operations are identified in files with the following extensions `[.graphql,](https://www.graphql-tools.com/docs/schema-loading#graphql-file-loader)[.ts, .tsx, .js, jsx](https://www.graphql-tools.com/docs/schema-loading#code-file-loader)` (`-o` option).
+
+### CLI output
+
+The `[@graphql-validate/cli](https://github.com/saihaj/graphql-validate)` CLI tool will output the GraphQL operations errors as follows:
```

Comment:
Consider replacing 
... will output the GraphQL operations errors ...
with
... will output any GraphQL operations errors ...